hoot-core/src/main/cpp/hoot/core/conflate/network/ConflictsNetworkMatcher.cpp

```diff
@@ -729,6 +729,7 @@ void ConflictsNetworkMatcher::_seedEdgeScores()
 
 void ConflictsNetworkMatcher::_printEdgeMatches()
 {
+  stringstream ss;
```

Comment:
Build up the output in a stream and then output it at the end in one `LOG_*` call.